__tests__/todoList.spec.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { render, fireEvent } from '@testing-library/react-native';
+import { userDataBuilder, itemDataBuilder } from 'dataBuilders';
+import { submitInput, submitMultiples } from 'submitInput';
+
+import Todo from '../src/screens/todo';
+
+const user = userDataBuilder();
+const todoItem = itemDataBuilder();
+
+test('all item should be toggle at the same time', () => {
```

Comment:
`'all item should be toggle at the same time'` perhaps you are missing the "when" they should be toggle at the same time

Reply:
Then it should be 'all item should be toggle at the same time when complete-all-button its press'